CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Bugfixes:
 
 Breaking Changes:
 - Microsoft.Extensions.Hosting related methods have been removed from the Castle.Windsor.Extensions.DependencyInjection package to the Castle.Windsor.Extensions.Hosting package (@ikkentim, #625, #628)
+- Obsolete components in Castle.Facilities.Logging have been removed. Extensions methods for built-in logging factories have been added. (@Jevonius, #616)
```

Comment:
Could you list out the public APIs that have been removed like we do in Castle Core (https://github.com/castleproject/Core/blob/master/CHANGELOG.md#440-2019-04-05) so it makes it clear to upgraders and to enable search. You don't need every enum member, just the enum type name.

Reply:
Here are the removed classes and methods.  Is anything else needed to merge this and create a release.  I am happy to help get this out the door.

- Removed class Castle.Facilities.Logging.LoggerImplementation
- Removed constructor Castle.Facilities.Logging.LoggingFacility(LoggerImplementation loggingApi)
- Removed constructor Castle.Facilities.Logging.LoggingFacility(LoggerImplementation loggingApi, string configFile)
- Removed constructor Castle.Facilities.Logging.LoggingFacility(string customLoggerFactory, string configFile)
- Removed method Castle.Facilities.Logging.LoggingFacility.LogUsing(LoggerImplementation loggingApi)
- Removed method Castle.Facilities.Logging.LoggingFacility.UseLog4Net()
- Removed method Castle.Facilities.Logging.LoggingFacility.UseLog4Net(string configFile)
- Removed method Castle.Facilities.Logging.LoggingFacility.UseNLog()
- Removed method Castle.Facilities.Logging.LoggingFacility.UseNLog(string configFile)